packages/hw-app-eth/src/Eth.js

```diff
@@ -240,6 +234,35 @@ export default class Eth {
       toSend.push(buffer);
       offset += chunkSize;
     }
+
+    rlpTx = ethers.utils.RLP.decode("0x" + rawTxHex);
```

Comment:
this makes signTransactionLargeChainID2 test fail with "data array too short".
Should we catch silently this case or does the test needs to be fixed ?

Reply:
also just before this part, we modify rawTx in case its an EIP155 transaction.
Should we use the updated rawTx instead of the original rawTxHex?

Reply:
no idea for `signTransactionLargeChainID2`
But I'd say yes, you should use the updated rawTx here

Reply:
rawTx is actually not updated so its ok

Reply:
signTransactionLargeChainID2 fixed